### ISSUE TITLE
Create Billing Plan returns 'The resource validation failed'

### ISSUE THREAD

**Issue by WaldemarCoding** [MicrosoftDocs/vsonline#452]
When I try to create a Billing Plan I get an 'The resource validation failed' error.

![image](https://user-images.githubusercontent.com/17712059/80916053-941f3b00-8d56-11ea-8fb5-52f4548b52c1.png)

If I go to https://portal.azure.com I see that resource groups have been created though. But they have no resource records inside.
![image](https://user-images.githubusercontent.com/17712059/80916086-dcd6f400-8d56-11ea-976c-5b56a85163ae.png)

**Comment by WaldemarCoding** [comment]
Removed all RG except for one.

**Comment by jkeech** [comment]
Hi @WaldemarCoding, the error message isn't great, but it looks like you were being blocked from creation new plans because the subscription was at the [max number of plans](https://docs.microsoft.com/en-us/visualstudio/online/resources/faq#what-are-visual-studio-onlines-quotas). Deleting the other RGs ended up deleting other billing plans and brought you below the limit, which allowed new plan creation to succeed.

After you create a billing plan, it will be empty initially. From there, you can create Codespaces inside of the plan. Let us know if the Codespace creation has any problems.

**Comment by WaldemarCoding** [comment]
Hi @jkeech 
maybe my comment was a little missleading, sorry. I removed all the previous RGs and tried it again.
There is no success message and the plan doesn't appear in the 'Create new Plan'-Dropdown even though it exists in azure.

If I got your [How-To](https://docs.microsoft.com/en-us/visualstudio/online/how-to/browser) right it should be listed in the dropdown after that and I should be able to create a codespace from withing this webpage, shouldn't I?
https://online.visualstudio.com/environments

This is what it looks like for me:
![VSO](https://user-images.githubusercontent.com/17712059/80921882-8929d200-8d79-11ea-912d-64e71475d8a9.gif)

**Comment by vso-ppe1** [comment]
Thanks for the details. That behavior is definitely not expected. It looks to me like the plan is being created, but it’s not correctly associating with your identity (since today plans are limited to only the user who creates them). Can you keep that plan around while we investigate on the backend? Cc @cmbrose

**Comment by jkeech** [comment]
Thanks for the details. That behavior is definitely not expected. It looks to me like the plan is being created, but it’s not correctly associating with your identity (since today plans are limited to only the user who creates them). Can you keep that plan around while we investigate on the backend? Cc @cmbrose

**Comment by WaldemarCoding** [comment]
Of course, no problem.

**Comment by JonathanAsbury-ACS** [comment]
Same problem here.  I also tried create the environment from the VS Code Extension and received the following: 

> Failed to find the entity that initialized the sign-in request.

Is there a PowerShell Script we can try?

**Comment by cmbrose** [comment]
@WaldemarCoding I believe I've found the cause here which is a bug in our user lookups - we are discussing a fix internally, but I will see if there is anything that I can do to get you unblocked faster.

@JonathanAsbury-ACS that sounds like a slightly different issue since it is related to creating environments.  Can you confirm whether or not you are able to create a plan and see it in the portal?

**Comment by cmbrose** [comment]
@WaldemarCoding I've done an adhoc update which should unblock you - can you try again?  You should be able to see all your existing plans now in the VSCS portal, but if not please also try creating one again.  Thanks!

**Comment by WaldemarCoding** [comment]
@cmbrose 
Can confirm it works now. Also didn't have to create a new plan, the last one created was available and preselected in the dropdown. Thank you :-)

**Comment by cmbrose** [comment]
Awesome! I'm going to go ahead and close this one!

@JonathanAsbury-ACS feel free to open another issue if you're still having problems
